lib/inspec/cli.rb

```diff
@@ -84,9 +84,13 @@ def json(target)
   desc 'check PATH', 'verify all tests at the specified PATH'
   option :format, type: :string
   profile_options
-  def check(path) # rubocop:disable Metrics/AbcSize
+  def check(path) # rubocop:disable Metrics/AbcSize,Metrics/MethodLength
     o = config
     diagnose(o)
+    o['log_location'] = STDERR if o['format'] == 'json'
```

Comment:
should this not be a `||=`? Is it possible to do json format and define where you want your logging to go?

Reply:
Also, shouldn't logging always default to stderr?

Reply:
Our logging is inconsistent. This PR improves the situation slightly by having the `check` command call the common log configuration method.  

Fixing the logging throughout is #3648 and is outside the scope of this PR.

Reply:
Agreed about the ||=